models/public/index.md

```diff
@@ -128,6 +128,7 @@ SSD-based and provide reasonable accuracy/performance trade-offs.
 | SSD with ResNet-50 V1 FPN | [TensorFlow\*](./ssd_resnet50_v1_fpn_coco/ssd_resnet50_v1_fpn_coco.md) | ssd_resnet50_v1_fpn_coco | 178.6807 | 59.9326 |
 | SSD with ResNet 34 1200x1200 | [PyTorch\*](./ssd-resnet34-1200-onnx/ssd-resnet34-1200-onnx.md)  | ssd-resnet34-1200-onnx | 433.411 | 20.058  |
 | RetinaFace-R50            | [MXNet\*](./retinaface-resnet50/retinaface-resnet50.md) | retinaface-resnet50 | 100.8478 | 29.427 |
+| RetinaFace Anti Covid     | [MXNet\*](./retinaface-anti-covid/retinaface-anti-covid.md)  | retinaface-anti-covid | 2.7781 | 0.5955 |
```

Comment:
I think, you should add this model to section:
```
## Object attributes
### Facial Landmarks
```
And `RetinaFace-R50` too.
cc @eaidova @vladimir-dudnik

Reply:
Since nobody opposed my suggestion, I ask you to add this model to specified subsection (you free to rename it, if necessary).

Reply:
Done